From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Tue, 8 Jun 2021 15:52:50 -0700
Subject: [PATCH] Workaround for inability to run autoreconf

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 m4/lib-link.m4 | 2 --
 1 file changed, 2 deletions(-)

diff --git a/m4/lib-link.m4 b/m4/lib-link.m4
index 9bdaffa..40fda0e 100644
--- a/m4/lib-link.m4
+++ b/m4/lib-link.m4
@@ -108,8 +108,6 @@ dnl   acl_hardcode_direct,
 dnl   acl_hardcode_minus_L.
 AC_DEFUN([AC_LIB_RPATH],
 [
-  dnl Tell automake >= 1.10 to complain if config.rpath is missing.
-  m4_ifdef([AC_REQUIRE_AUX_FILE], [AC_REQUIRE_AUX_FILE([config.rpath])])
   AC_REQUIRE([AC_PROG_CC])                dnl we use $CC, $GCC, $LDFLAGS
   AC_REQUIRE([AC_LIB_PROG_LD])            dnl we use $LD, $with_gnu_ld
   AC_REQUIRE([AC_CANONICAL_HOST])         dnl we use $host
